src/Pages/kcy/Main/Feed.js

```diff
@@ -0,0 +1,227 @@
+import React from "react";
+import feedHeaderData from "./Data/FeedHeaderData";
+import Comment from "./Comment";
+import "../config/Feed.scss";
+class Feed extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      feedHeaderImage: feedHeaderData,
+      inputValue: "",
+      inputValueList: [],
+      num: 1,
+      mode: false,
+      modalMode: false,
+    };
+  }
+
+  /* slide */
+
+  /* modal */
+  modal = () => {
+    this.setState({
+      modalMode: !this.state.modalMode,
+    });
+  };
+
+  /* comment Heart Btn  */
+
+  heartChange = (mode, index) => {
+    const changedList = [...this.state.inputValueList];
+    changedList[index].mode = !mode;
+
+    this.setState({
+      inputValueList: changedList,
+    });
+  };
+
+  /* Feed Heart Btn */
+
+  list = (e) => {
+    if (e.target) {
+      this.setState({
+        mode: !this.state.mode,
+      });
+    }
+  };
+
+  /* comment create & remove  */
+
+  createItem = (e) => {
+    this.setState({
+      inputValue: e.target.value,
+    });
+  };
+
+  removeItem = (id) => {
+    const { inputValueList } = this.state;
+
+    this.setState({
+      inputValueList: inputValueList.filter((e) => e.id !== id),
+    });
+  };
+
+  inputCreate = () => {
+    this.setState({
+      inputValueList: this.state.inputValueList.concat({
+        id: this.state.num,
+        comment: this.state.inputValue,
+        mode: false,
+      }),
+      inputValue: "",
+      num: this.state.num + 1,
+    });
+  };
```

Comment:
```suggestion
  inputCreate = () => {
    const { num, inputValueList } = this.state
    const addInputList = [...inputValueList, {
      id: num,
      comment: inputValue,
      mode: false,
    }]
    
    this.setState({
      inputValueList: addInputList,
      inputValue: "",
      num: num + 1,
    });
  };
```
concat 대신 스프레드 연산자 사용해줄 수 있습니다. 검색해보시고 어떻게 사용하는지 댓글에 달아주세요!